dev/release.sh

```diff
@@ -0,0 +1,614 @@
+#! /bin/sh -e
+# Copyright 2020 The OpenSSL Project Authors. All Rights Reserved.
+#
+# Licensed under the Apache License 2.0 (the "License").  You may not use
+# this file except in compliance with the License.  You can obtain a copy
+# in the file LICENSE in the source distribution or at
+# https://www.openssl.org/source/license.html
+
+usage () {
+    cat <<EOF
```

Comment:
Don't you want ```\EOF``` or similar to avoid quote-eating?

Reply:
Did you actually check that quotes are eaten?  Here's what `./dev/release.sh --help` displayed to me:

```
: ; ./dev/release.sh --help
Usage: release.sh [ options ... ]

--alpha         Start or increase the "alpha" pre-release tag.
--next-beta     Switch to the "beta" pre-release tag after alpha release.
                It can only be given with --alpha.
--beta          Start or increase the "beta" pre-release tag.
--final         Get out of "alpha" or "beta" and make a final release.
                Implies --branch.

--branch        Create a release branch 'openssl-{major}.{minor}.x',
                where '{major}' and '{minor}' are the major and minor version
                numbers.

--no-upload     Don't upload to upload@dev.openssl.org.
--no-update     Don't perform 'make update'.
--verbose       Verbose output.
--debug         Include debug output.  Implies --no-upload.

--force         Force execution

--help          This text
--manual        The manual

If none of --major, --minor, --alpha, --beta, or --final are given, this script
tries to figure out the next step.
```

Reply:
the joys of shells.  good.